h2o-core/src/main/java/water/api/LogsHandler.java

```diff
@@ -78,6 +78,14 @@ public void doIt() {
               log = "H2O logging not configured.";
             }
             break;
+          case "telemetry":
```

Comment:
This is for case our default Telemetry Extension which outputs reports to a file